src/sshping.cxx

```diff
@@ -531,16 +625,21 @@ int run_upload_test(ssh_session ses) {
         return rc;
     }
 
-    struct timespec t2;
-    clock_gettime(CLOCK_MONOTONIC, &t2);
+	uint64_t t2 = GetTime();
 
     char buf[MEGA];
     srand(getpid());
     for (size_t i=0; i < sizeof(buf); i++) {
         buf[i] = (rand() & 0x3f) + 32;
     }
     for (int i=0; i < size; i++) {
-        rc = ssh_scp_push_file(scp, remfile, MEGA, S_IRWXU);
+        rc = ssh_scp_push_file(scp, remfile, MEGA, 
+#ifdef _WIN32 
+			448
```

Comment:
Anonymous constants are generally frowned upon.  Just define S_IRWXU (or whatever the 448 represents) above in one of the windows specific sections; that'll improve readability too.

Reply:
Yeah this parameter is for Unix permissions. S_IRWXU is 448 in decimal (700 in octal). Not 100% sure if this is correct for windows too. But maybe the library will handle this correctly.

```
#ifdef _WIN32
        const int mode = S_IRWXU;
#else
        const int mode = 448;
#endif

        rc = ssh_scp_push_file(scp, remfile, MEGA, mode);
```

Something like this should be fine I guess?